vignettes/pk-analysis.Rmd

```diff
@@ -90,3 +94,85 @@ exportPKAnalysesToCSV(pkAnalyses = pkAnalysis, filePath = csvPKAnalysisPath)
 # Load from csv
 pkAnalysisLoaded  <- importPKAnalysesFromCSV(filePath = csvPKAnalysisPath, simulation = sim)
 ```
+
+
+## User-Defined PK Parameters
+
+The `{ospsuite}` package also supports user-defined PK Parameters, e.g. PK Parameter that can be tailored to specific project needs.
+This feature is useful when calculating PK Parameters for specific time intervals, or to apply PK Parameters for output not using 
+the dimension `Concentration`. 
+
+A User-Defined PK Parameter is always based on an existing PK-Parameter and simply extends the way the output is calcualted.
+It is not possible at the moment to define your own `formula`.
+
+The utility `addUserDefinedPKParameter` creates and returns an instance of the `UserDefinedPKParameter` which can then be parameterized
+for specific requirements.
```

Comment:
Here we could directly link to the online help:
The utility [`addUserDefinedPKParameter`](https://www.open-systems-pharmacology.org/OSPSuite-R/reference/addUserDefinedPKParameter.html) creates and returns an instance of the [`UserDefinedPKParameter`](https://www.open-systems-pharmacology.org/OSPSuite-R/reference/UserDefinedPKParameter.html) which can then be parameterized for specific requirements.